src/topp/OpenSwathChromatogramExtractor.cpp

```diff
@@ -52,11 +52,11 @@ using namespace std;
 #include <omp.h>
 #endif
 
-#ifdef _OPENMP
-  #define IF_MASTERTHREAD if (omp_get_thread_num() ==0)  
-#else
-  #define IF_MASTERTHREAD 
-#endif    
+// #ifdef _OPENMP
```

Comment:
I know, its copied from MSSim :)

but the place where its was used is commented out, so why keep either of 
them?

Am 4/22/2015 um 19:16 schrieb Hannes Roest:
>
> In src/topp/OpenSwathChromatogramExtractor.cpp 
> <https://github.com/OpenMS/OpenMS/pull/1369#discussion_r28894163>:
>
> > @@ -52,11 +52,11 @@ using namespace std;
> >  #include <omp.h>
> >  #endif
> >
> > -#ifdef _OPENMP
> > -  #define IF_MASTERTHREAD if (omp_get_thread_num() ==0)
> > -#else
> > -  #define IF_MASTERTHREAD
> > -#endif
> > +// #ifdef _OPENMP
>
> the code is correct and useful for debugging purposes
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OpenMS/OpenMS/pull/1369/files#r28894163>.
>